packages/core/src/utils.ts

```diff
@@ -530,8 +530,18 @@ export function isObservable<T>(value: any): value is Subscribable<T> {
 }
 
 export const symbolObservable = (() =>
-  (typeof Symbol === 'function' && (Symbol as any).observable) ||
-  '@@observable')();
+  (typeof Symbol === 'function' && Symbol.observable) || '@@observable')();
+
+export const interopSymbols = {
+  [symbolObservable]: function () {
+    return this;
+  },
+  // this gets stripped by Babel to avoid having "undefined" property in environments without this non-standard Symbol
+  // it has to be here to be included in the generated .d.ts
```

Comment:
have you confirmed that this works? is the `Symbol.observable` emitted in the `.d.ts`? is this property in the dist files from here by Babel?

Reply:
Yes, only `[symbolObservable]` is used in the ES file, and `[Symbol.observable]` is present in the type declarations.